docs/book/mlops_stacks/orchestrators/airflow.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Orchestrate pipelines with Airflow
+---
+
+The Airflow orchestrator is an [orchestrator](./overview.md) flavor provided with
+the ZenML `airflow` integration that uses [Airflow](https://airflow.apache.org/)
+to run your pipelines.
+
+## When to use it
+
+You should use the Airflow orchestrator if
+* you're already using Airflow
+* you want to run your pipelines locally with a more production-ready setup 
+than the [local orchestrator](./local.md)
+
+If you're looking to run your pipelines in the cloud, take a look at other
+[orchestrator flavors](./overview.md#orchestrator-flavors).
+
+{% hint style="info" %}
+
+We're currently reworking the Airflow orchestrator to make sure it works
+not only locally but also with Airflow instances deployed on cloud infrastructure.
+
+{% endhint %}
+
+## How to deploy it
+
+The Airflow orchestrator works without any additional infrastructure setup.
+
+## How to use it
+
+To use the Airflow orchestrator, we need:
+* The ZenML `airflow` integration installed. If you haven't done so, run 
+    ```shell
+    zenml integration install airflow
+    ```
+
+We can then register the orchestrator and use it in our active stack:
+```shell
+zenml orchestrator register <NAME> \
+    --flavor=airflow
+
+# Add the orchestrator to the active stack
+zenml stack update -o <NAME>
```

Comment:
Not sure either. I put this on all my pages, as otherwise I'd have to either register a complete stack
or have an incomplete command. Happy to change it though if you think a different way is better

Reply:
What's wrong with `stack update`? I use it all the time. Super useful.